lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -144,13 +144,14 @@ def verify(creditcard, options = {})
           exp_month = creditcard.month.to_s
           exp_year = creditcard.year.to_s
           expiration = "#{exp_month}/#{exp_year}"
+          zip = options[:billing_address][:zip] || nil
           payload = {
             credit_card: {
               number: creditcard.number,
               expiration_date: expiration,
               cvv: creditcard.verification_value,
               billing_address: {
-                postal_code: options[:billing_address][:zip]
+                postal_code: zip
```

Comment:
I think you might be able to achieve what we need here without instantiating `zip` above - you could instead do `options[:billing_address].try(:[], :zip)`